src/components/alert/alert.vue

```diff
@@ -95,27 +100,47 @@ export default {
   },
 
   computed: {
+    baseClasses() {
```

Comment:
@SergioCrisostomo 
This is what I wanted you to look at. I made a constructor method to distribute the right classnames. Might export this to a mixin if I want to use it in more files.

Reply:
I like the idea. Since the `prefix` is not a instance property I think we could have this as a function we `import`. A mixin is good als, but would be more clear to import the function.

And since the 2 functions are so related we could do:

```
const {classes, prefix} = this.classesConstructor('icon', false);
```